vscode/README-DEV.md

```diff
@@ -1,9 +1,9 @@
-# [DEV BUILD] Azure Quantum Development Kit
+# [DEV BUILD] Microsoft Quantum Development Kit
 
-This is a development build of the Azure Quantum Development Kit, and is intended for early testing
+This is a development build of the Microsoft Quantum Development Kit, and is intended for early testing
 of upcoming features.
 
 To install the tested version of this extension, visit <https://marketplace.visualstudio.com/items?itemName=quantum.qsharp-lang-vscode>.
 The README for the stable build contains details on features and usage.
 
-For more information about the Modern QDK and Azure Quantum, visit [https://aka.ms/AQ/Documentation](https://aka.ms/AQ/Documentation).
+For more information about the Modern QDK and Microsoft Quantum, visit [https://aka.ms/qdk](https://aka.ms/qdk).
```

Comment:
How do we feel about the "Modern QDK" now?

Reply:
Good catch - we hate it! Will delete and look for other occurence :-)